maestro-cli/src/main/java/maestro/cli/command/StartDeviceCommand.kt

```diff
@@ -87,13 +87,99 @@ class StartDeviceCommand : Callable<Int> {
             if (parts.size == 2) {
                 val language = parts[0]
                 val country = parts[1]
+
+                validateLocale(language, country)
+
                 return Pair(language, country)
             } else {
-                throw CliError("Wrong device locale format was provided $it. A combination of lowercase ISO-639-1 code and uppercase ISO-3166-1 code should be used, i.e. \"de_DE\" for Germany")
+                throw CliError("Wrong device locale format was provided $it. A combination of lowercase ISO-639-1 code and uppercase ISO-3166-1 code should be used, i.e. \"de_DE\" for Germany. More info can be found here https://maestro.mobile.dev/")
             }
         }
 
         // use en_US locale as a default
         return Pair("en", "US")
     }
+
+    private fun validateLocale(language: String, country: String) {
+        if (!SUPPORTED_LANGUAGES.map { it.first }.contains(language)) {
+            val languages = SUPPORTED_LANGUAGES.joinToString("\n")
+            throw CliError("$language language is currently not supported by Maestro, please check that it is a valid ISO-639-1 code. Here is a full list of supported languages:\n" +
+                    "\n" +
+                    languages
+            )
+        }
+        if (!SUPPORTED_COUNTRIES.map { it.first }.contains(country)) {
+            val countries = SUPPORTED_COUNTRIES.joinToString("\n")
+            throw CliError("$country country is currently not supported by Maestro, please check that it is a valid ISO-3166-1 code. Here is a full list of supported countries:\n" +
+                    "\n" +
+                    countries
+            )
+        }
+    }
+
+    companion object {
+        // ISO-639-1
+        private val SUPPORTED_LANGUAGES = listOf(
```

Comment:
Did we verify that this list is the same for both Android and iOS?

Reply:
answered offline